core/app/models/spree/order.rb

```diff
@@ -73,10 +73,9 @@ class CannotRebuildShipments < StandardError; end
     # Customer info
     belongs_to :user, class_name: Spree::UserClassHandle.new, optional: true
     belongs_to :bill_address, foreign_key: :bill_address_id, class_name: 'Spree::Address', optional: true
-    alias_attribute :billing_address, :bill_address
-
+    alias_method :billing_address, :bill_address
```

Comment:
Unfortunately, this doesn't fully match the existing behavior. For example, previously we could do  (and we do it [here](https://github.com/solidusio/solidus/blob/682033394bb1cec88ef6da58223710b870b5dc15/sample/db/samples/orders.rb#L17), for example) 
```ruby
Spree::Order.new(billing_addres: Spree::Address.new)
``` 

but now this raises `ActiveModel::UnknownAttributeError: unknown attribute 'billing_address' for Spree::Order. (ActiveModel::UnknownAttributeError)`

Reply:
adding `alias_method :billing_address=, :bill_address=` should fix this issue